.solcover.js

```diff
@@ -14,6 +14,10 @@ module.exports = {
     'packages/oz/upgradeability/ContextUpgradeSafe.sol',
     'packages/oz/upgradeability/ERC20Initializable.sol',
     'packages/oz/upgradeability/Initializable.sol',
+    'packages/openzeppelin-upgradeability/OwnedUpgradeabilityProxy.sol',
+    'packages/openzeppelin-upgradeability/Proxy.sol',
+    'packages/openzeppelin-upgradeability/UpgradeabilityProxy.sol',
+    'packages/openzeppelin-upgradeability/VersionedInitializable.sol',
```

Comment:
What's the differences between `oz/upgradeability` and `openzeppelin-upgradeability`? I think we can just use the same folder.